tests/nn/data_parallel/test_sharded_ddp_features.py

```diff
@@ -22,7 +22,6 @@
 from fairscale.utils.testing import (
```

Comment:
no changes in this file except from
- removing duplicated test with the other sharded_ddp test file (probably a botched merge)
- removing gloo tests

Reply:
test removal has been moved to another PR, now just gloo backend removal as expected